Sourcery/Parsing/Utils/Verifier.swift

```diff
@@ -16,10 +16,16 @@ enum Verifier {
         case approved
     }
 
-    public static func canParse(content: String) -> Result {
+    public static func canParse(content: String,
+                                path: Path,
+                                parsableExtensions: [String]) -> Result {
         guard !content.isEmpty else { return .approved }
 
-        if content.hasPrefix(Sourcery.generationMarker) {
+        let hasParsableExtension = (parsableExtensions.filter({ (ext) -> Bool in
+            return path.hasExtension(as: ext)
+        }).isEmpty == false)
+
+        if content.hasPrefix(Sourcery.generationMarker) && hasParsableExtension == false {
```

Comment:
Why not just use a boolean flag to signal that generated files should be parsed?

Reply:
Cause not all generated files needs to be parsed, I want to selected only some files to be parsed in the second pass

Reply:
I think if you just said 'parse generated files' it might lead to chicken and egg scenario, I prefer this approach as its more explicit for the end-user